src/components/Researcher/ActivityList/GameCreator.tsx

```diff
@@ -50,65 +50,63 @@ export default function GameCreator({
   const defaultIntertrialDuration = 0.5
   const defaultBubbleDuration = 1.0
-  const [settings, setSettings] = useState(
-    !!value
-      ? value?.settings
-      : (value?.spec && ["lamp.jewels_a", "lamp.jewels_b"].includes(value.spec)) ||
-        ["lamp.jewels_a", "lamp.jewels_b"].includes(activitySpecId)
-      ? {
-          mode: 1,
-          variant:
-            activitySpecId === "lamp.jewels_a" || (value?.spec && value?.spec === "lamp.jewels_a")
-              ? "trails_a"
-              : "trails_b",
-          beginner_seconds: 90,
-          intermediate_seconds: 30,
-          advanced_seconds: 25,
-          expert_seconds: 15,
-          diamond_count: 15,
-          shape_count: (value?.spec && "lamp.jewels_b" === value.spec) || "lamp.jewels_b" == activitySpecId ? 2 : 1,
-          bonus_point_count: 50,
-          x_changes_in_level_count: 1,
-          x_diamond_count: 0,
-          y_changes_in_level_count: 1,
-          y_shape_count: 1,
-        }
-      : ["lamp.balloon_risk"].includes(activitySpecId)
-      ? {
-          balloon_count: defaultBallonCount,
-          breakpoint_mean: defaultMean,
-          breakpoint_std: defaultSD,
-        }
-      : ["lamp.pop_the_bubbles"].includes(activitySpecId)
-      ? {
-          bubble_count: defaultBubbleCount,
-          bubble_speed: defaultBubbleSpeed,
-          intertrial_duration: defaultIntertrialDuration,
-          bubble_duration: defaultBubbleDuration,
-        }
-      : ["lamp.scratch_image"].includes(activitySpecId)
-      ? { threshold: 80 }
-      : {}
-  )
+  const settings = !!value
+    ? value?.settings
+    : (value?.spec && ["lamp.jewels_a", "lamp.jewels_b"].includes(value?.spec)) ||
+      ["lamp.jewels_a", "lamp.jewels_b"].includes(activitySpecId)
+    ? {
+        mode: 1,
+        variant:
+          activitySpecId === "lamp.jewels_a" || (value?.spec && value?.spec === "lamp.jewels_a")
+            ? "trails_a"
+            : "trails_b",
+        beginner_seconds: 90,
+        intermediate_seconds: 30,
+        advanced_seconds: 25,
+        expert_seconds: 15,
+        diamond_count: 15,
+        shape_count: (value?.spec && "lamp.jewels_b" === value.spec) || "lamp.jewels_b" == activitySpecId ? 2 : 1,
+        bonus_point_count: 50,
+        x_changes_in_level_count: 1,
+        x_diamond_count: 0,
+        y_changes_in_level_count: 1,
+        y_shape_count: 1,
+      }
+    : ["lamp.balloon_risk"].includes(activitySpecId)
+    ? {
+        balloon_count: defaultBallonCount,
+        breakpoint_mean: defaultMean,
+        breakpoint_std: defaultSD,
+      }
+    : ["lamp.pop_the_bubbles"].includes(activitySpecId)
+    ? {
+        bubble_count: defaultBubbleCount,
+        bubble_speed: defaultBubbleSpeed,
+        intertrial_duration: defaultIntertrialDuration,
+        bubble_duration: defaultBubbleDuration,
+      }
+    : ["lamp.scratch_image"].includes(activitySpecId)
+    ? { threshold: 80 }
+    : {}
```

Comment:
Can you remove all of these? They should no longer be required and are a part of the JSONSchema.

Reply:
@avaidyam Removed those settings.